javascript/javascript1/week3/warmuptask.js

```diff
@@ -0,0 +1,70 @@
+//Item array removal
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+function removeTheName(name) {
+  return name !== nameToRemove;
+}
+const newArray = names.filter(removeTheName);
+console.log(newArray);
+
+//When will we be there??
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+function calcTravelTime(travelInfo) {
+  let time = travelInfo["destinationDistance"] / travelInfo["speed"];
```

Comment:
`const` unless you will re-assign it! Also, using the . annotation here as mentioned above

Reply:
changed to const